ui/app/components/oidc/assignment-form.js

```diff
@@ -43,7 +46,7 @@ export default class OidcAssignmentFormComponent extends Component {
       }
     } catch (error) {
       const message = error.errors ? error.errors.join('. ') : error.message;
-      this.flashMessages.danger(message);
+      this.errorBanner = message;
```

Comment:
Yeah, I'm all about this. I had talked to Jordan about my concerns "flashing" error messages instead of banners. My thoughts as a user is I'd want the error to stay static so I can screen shot, whatever, instead of flashing on the screen.

Reply:
Yeah - I agree!